Makefile

```diff
@@ -15,25 +17,34 @@ test:
 .PHONY: demo
 demo: petstore
 
+compile:
```

Comment:
let's not make another target that generates a file - it would need to be grouped with associated actions